playground/frontend/playground_components/lib/dismissible_overlay.dart

```diff
@@ -0,0 +1,25 @@
+import 'package:flutter/material.dart';
+
+class DismissibleOverlay extends StatelessWidget {
+  final void Function() close;
+  final Positioned child;
+
+  const DismissibleOverlay({
+    required this.close,
+    required this.child,
+  });
+
+  @override
+  Widget build(BuildContext context) {
+    return Stack(
+      children: [
+        Positioned.fill(
+          child: GestureDetector(
```

Comment:
What about the mouse pointer? If we need it to change, I like combining `GestureDetector` with `MouseRegion` for that.

Reply:
I'll clarify if it's an expected behavior.

Reply:
Oleh: "I don't think it should close when hovering mouse outside"